test/integration/connection-monitoring-and-pooling/connection_monitoring_and_pooling.spec.test.ts

```diff
@@ -52,3 +53,19 @@ describe('Connection Monitoring and Pooling Spec Tests (Integration)', function
     )
   });
 });
+
+describe('Connection Monitoring and Pooling Spec Tests (Integration) - logging', function () {
+  const tests = loadSpecTests('connection-monitoring-and-pooling', 'logging');
+
+  runUnifiedSuite(tests, (test: { description: string }) => {
+    if (
+      [
+        'waitQueueMultiple should be included in connection pool created message when specified',
+        'waitQueueSize should be included in connection pool created message when specified'
+      ].includes(test.description)
+    ) {
+      return 'not applicable: waitQueueSize not supported';
+    }
```

Comment:
I am open to filing a new ticket to skip the `'Connection checkout fails due to error establishing connection'` spec test since this seems to be failing due to an existing event-ordering issue in the driver and not the logger itself.